src/components/utah-design-system/Accordion.jsx

```diff
@@ -0,0 +1,37 @@
+import * as Accordion from '@radix-ui/react-accordion';
+import PropTypes from 'prop-types';
+
+export function AccordionRoot({ type, children }) {
+  return <Accordion.Root type={type}>{children}</Accordion.Root>;
+}
+
+AccordionRoot.propTypes = {
+  type: PropTypes.oneOf(['single', 'multiple']),
+  children: PropTypes.node.isRequired,
+};
+
+export function AccordionPanel({ title, children }) {
+  return (
+    <Accordion.Item value={title} className="mb-1 overflow-hidden rounded-md">
+      <Accordion.Header className="rounded-sm">
+        <Accordion.Trigger className="group w-full justify-between rounded-b-none rounded-t-md border-[--primary-color] bg-[--primary-color] px-2 py-1 text-left text-lg leading-5 text-white active:transform-none active:shadow-none">
+          {title}
+          <span
+            className="utds-icon-before-circle-chevron-down flex content-center justify-center text-base text-white transition before:mr-0 before:content-['L'] group-data-[state=open]:rotate-180"
+            aria-hidden="true"
+          />
+        </Accordion.Trigger>
+      </Accordion.Header>
+      <Accordion.Content className="overflow-hidden data-[state=closed]:animate-slide-up data-[state=open]:animate-slide-down">
+        <div className="rounded-b-md border-b border-l border-r border-gray-300 p-2">
```

Comment:
**polish**: I know the design system hasn't created the guidelines for the accordion yet but based on the mockups, I wonder if a `bg-gray-50` would be nice to separate the background color from the panel color and a little more `px-*` padding of the child content.

![image](https://user-images.githubusercontent.com/325813/234640109-df1b7d09-3cb9-4dc2-8014-6122dda4dec3.png)

Reply:
2355ce1e79e117bcf4cff3dc12f48dc34cdc388f